client/src/app/App.js

```diff
@@ -1809,155 +1810,159 @@ export class App extends PureComponent {
 
         <div className={ css.App }>
 
-          <SlotFillRoot>
+          <KeyboardInteractionTrapContext.Provider value={ this.triggerAction }>
```

Comment:
Maybe I'm missing the point here but is there a reason why we don't only wrap PluginsRoot component? AFAIK KeyboardInteractionTrap was meant to be used by plugins.

Reply:
It's meant to be used by modals, regardless of whether they are embedded in plugins or not. The benefit of the `KeyboardInteractionTrap` is that you don't accidentally activate editor's shortcuts while a modal is open. Also, there is still one modal which has not been moved to the plugins yet: KeyboardShortcuts. Therefore, I don't see reason to limit the scope of the context.